core/backend/src/IModelHost.ts

```diff
@@ -491,6 +491,7 @@ export class IModelHost {
     this.configuration = options;
     this.setupTileCache();
 
+    // eslint-disable-next-line @typescript-eslint/unbound-method
     process.once("beforeExit", IModelHost.shutdown);
```

Comment:
```suggestion
    process.once("beforeExit", () => IModelHost.shutdown());
```

I don't understand why you would go to the trouble of disabling this rule all over the place instead of just conforming to it.

Reply:
I didn't wanna inadvertently break the tests or anything so I just took the disabling route when I started, I can push the confirming version instead. Thanks.

Reply:
done